src/modules/ekf2/EKF/aid_sources/gravity/gravity_fusion.cpp

```diff
@@ -54,8 +54,9 @@ void Ekf::controlGravityFusion(const imuSample &imu)
 
 	const float upper_accel_limit = CONSTANTS_ONE_G * 1.1f;
 	const float lower_accel_limit = CONSTANTS_ONE_G * 0.9f;
-	const bool accel_lpf_norm_good = (_accel_magnitude_filt > lower_accel_limit)
-					 && (_accel_magnitude_filt < upper_accel_limit);
+	const float accel_lpf_norm_sq = _accel_lpf.getState().norm_squared();
```

Comment:
`_accel_lpf` is only being updated during init, but I was thinking we should move it regardless.

https://github.com/PX4/PX4-Autopilot/blob/beb7e7d79b7d0dd15b7726855067e6fa612bbc70/src/modules/ekf2/EKF/ekf.cpp#L180-L198

Reply:
I was moving it here, but got a little sidetracked. https://github.com/PX4/PX4-Autopilot/pull/24046

Reply:
Ouch, I didn't realize that; thanks